package.json

```diff
@@ -63,7 +63,7 @@
     "browserify": "10.2.6",
     "browserify-shim": "3.8.9",
     "del": "1.2.0",
-    "ember-cli": "1.13.1",
+    "ember-cli": "1.13.8",
```

Comment:
Did you do any necessary upgrades to `bower.json`/`package.json` that this `ember-cli` upgrade needed?

It looks like the removal of `Brocfile.js` was not needed anymore due to the upgrade, is this correct?